Todo/JS/TodoServerSide/TodoServer_DL/RamDB.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using TodoServer_Commons;
+namespace TodoServer_DL
+{
+    public class RamDB : TodoServer_Commons.IDB
+    {
+        private List<TodoTask> _tasks;
+
+        public RamDB()
+        {
+            _tasks = new List<TodoTask>();
+        }
+        public IEnumerable<TodoTask> GetTasks()
+        {
+            return _tasks;
+        }
+
+        public bool AddTask(TodoTask addedTask)
+        {
+            _tasks.Add(addedTask);
+            return true;

```

Comment:
returning a bool for confirmation is bad practice, its better for it to be void and throw an exception if needed